inject.js

```diff
@@ -154,22 +154,25 @@
 
       this.div=this.initializeControls();
 
-      target.addEventListener('play', this.handlePlay = function(event) {
-        if (!tc.settings.rememberSpeed) {
-          if (!tc.settings.speeds[target.currentSrc]) {
-            tc.settings.speeds[target.currentSrc] = this.speed;
+      if (isRateChangeBlocked() === false) {
```

Comment:
This original listener was added to fix YouTube fullscreen from resetting the playback rate. Not sure why a YT specific bug has been applied globally, but in the case of pluralsight it has the opposite effect. For now, bypassing this fix seems to work fine.